### ISSUE TITLE
Is there a way to check tmux continuum is running? It's not restoring my sessions

### ISSUE THREAD

**Issue by bryanchen18** [tmux-plugins/tmux-continuum#54]
Do I need to grep or vim something to ensure its running?
```
 set -s escape-time 0
 set -g mouse on

 # Increase scroll-back limit
 set-option -g history-limit 5000

 # List of plugins
 set -g @plugin 'tmux-plugins/tpm'
 set -g @plugin 'tmux-plugins/tmux-sensible'
 set -g @plugin 'tmux-plugins/tmux-resurrect'
 set -g @plugin 'tmux-plugins/tmux-continuum'

 # Change to sane key bindings for pane-splitting.
 bind | split-window -h
 bind - split-window -v
 unbind '"'
 unbind %

 # Enable clicking on panes to select
 set -g mouse on

 # List of plugins
 set -g @plugin 'tmux-plugins/tpm'
 set -g @plugin 'nhdaly/tmux-scroll-copy-mode'

 # Other examples:
 # set -g @plugin 'github_username/plugin_name'
 # set -g @plugin 'git@github.com/user/plugin'
 # set -g @plugin 'git@bitbucket.com/user/plugin'

 # Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
 run '~/.tmux/plugins/tpm/tpm'

 set -g @continuum-restore 'on'
 set -g @continuum-boot 'on'
 set -g @continuum-boot-options 'iterm'
 set -g @continuum-save-interval '5'
 set -g status-right 'Continuum status: #{continuum_status}'
 set -g @resurrect-capture-pane-contents 'on'

 # Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
 run -b '~/.tmux/plugins/tpm/tpm'
```

**Comment by bruno-** [comment]
Hey, recent tmux versions made a change that caused a bug in this plugin. I'm just pushing an update that will fix this.

**Comment by bruno-** [comment]
Fixed with 4e2279688d8a577ed14f42c4e8af1acb58100a57
